ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -158,14 +182,26 @@ object AuthServiceJWTCodec {
       val scopes = scope.toList.collect({ case JsString(scope) => scope.split(" ") }).flatten
       // We're using this rather restrictive test to ensure we continue parsing all legacy sandbox tokens that
       // are in use before the 2.0 release; and thereby maintain full backwards compatibility.
-      if (scopes.contains(scopeLedgerApiFull))
+      val audienceValue = readOptionalStringOrSingletonArray(propAud, fields)
+      if (scopes.contains(scopeLedgerApiFull)) {
         // Standard JWT payload
         StandardJWTPayload(
-          participantId = readOptionalString("aud", fields),
+          participantId = audienceValue,
           userId = readOptionalString("sub", fields).get, // guarded by if-clause above
           exp = readInstant("exp", fields),
+          format = StandardJWTTokenFormat.Scope,
         )
-      else {
+      } else if (audienceValue.exists(_.startsWith(audPrefix))) {
```

Comment:
Design doc does not say anything about this additional restriction that participantId for this format of the token is required then. Makes sense to me, it would also eliminate the problem of handling empty string vs non-provided `participantId` within this format

Reply:
apologies, I didn’t spell that out explicitly. I’ve updated the design doc to explicitly require the participant id.